lhotse/dataset/input_strategies.py

```diff
@@ -180,7 +180,34 @@ class AudioSamples(BatchIO):
     .. automethod:: __call__
     """
 
-    def __call__(self, cuts: CutSet) -> Tuple[torch.Tensor, torch.IntTensor]:
+    def __init__(
```

Comment:
Above where it says "It pads the recordings if needed" (line 178), would be nice to say why that might be needed.

Reply:
I extended the docs in that place and added some tests. Merging!